test/unit/exception_handling_test.rb

```diff
@@ -122,6 +122,17 @@ def self.notify(data)
     end
   end
 
+
+  context "#log_warning" do
+    should "have empty array as a backtrace" do
+      mock(ExceptionHandling).log_error(is_a(ExceptionHandling::Warning)) do |error|
+        assert_equal ExceptionHandling::Warning, error.class
```

Comment:
This `assert_equal` seems redundant with the `is_a` in the line above. Also, testing classes isn't good because it violates the Liskov Substitution Principle.  `is_a` conforms to the LSP because it allows derived classes too.  So...I think you should just delete this line.